### ISSUE TITLE
Sonarr on remote system.

### ISSUE THREAD

**Issue by morpheus2n2** [mdhiggins/sickbeard_mp4_automator#874]
Tring to pin down the problem so seeing if maybe there's something with this script I need to do.
Basically, I have Sonarr running on a remote Windows VM via my NAS, and I have Sabnzbd & Deluge running on my main PC as it has more grunt to run the script.
I am trying to get Sonarr to work with remote path mappings and wondering if its not working due to the script picking up the file first?
Is there a way to get it to work?

**Comment by mdhiggins** [comment]
Script will only pick up files first if you’re using completed download handling and a post downloader script at the same time (SAB/Deluge/uTorrent/NZBGet)

Just don’t use both as the same time

From: morpheus2n2
Sent: Friday, April 13, 2018 3:11 PM
To: mdhiggins/sickbeard_mp4_automator
Cc: Subscribed
Subject: [mdhiggins/sickbeard_mp4_automator] Sonarr on remote system. (#874)

Tring to pin down the problem so seeing if maybe there's something with this script I need to do.
Basically, I have Sonarr running on a remote Windows VM via my NAS, and I have Sabnzbd & Deluge running on my main PC as it has more grunt to run the script.
I am trying to get Sonarr to work with remote path mappings and wondering if its not working due to the script picking up the file first?
Is there a way to get it to work?
